client/knowledge_base_search/src/containers/ResultsContainer/ResultComparison.js

```diff
@@ -83,11 +99,21 @@ class ResultComparison extends Component {
           {
             full_result_index === index
               ? (
-                  <FullResult
-                    key={full_result_index + '_' + full_result_type}
-                    transitionTimeout={fullResultTransitionTimeout}
-                    {...this.getFullAnswer()}
-                  />
+                  this.state.full_answer === undefined ? (this.getFullAnswer()) : (
+                    this.state.full_answer.error ? (
+                      <ErrorContainer
+                        key={'error_container'}
+                        results_error={this.state.full_answer.error.message}
+                        enriched_results_error={this.state.full_answer.error.stack}
```

Comment:
so based on this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/prototype I don't think we can assume `stack` exists by default. I think we need to change the `query` action to return just the `error.message`